landingai/predict.py

```diff
@@ -127,19 +127,19 @@ def _configure_logger() -> None:
     requests_log.propagate = False
 
 
-def _extract_prediction(response: Dict[str, Any]) -> Sequence[Prediction]:
+def _extract_prediction(response: Dict[str, Any]) -> List[Prediction]:
     response_type = response["backbonetype"]
     if response_type is None and response["type"] == "SegmentationPrediction":
         response_type = "SegmentationPredictionVP"  # Visual Prompting response
     if response_type is None:
         response_type = response["type"]  # Classification response
     predictions = PREDICTION_EXTRACTOR[response_type](response)
-    return predictions
+    return predictions  # type: ignore
```

Comment:
i remember i changed the output type to `Sequence` to avoid this kind of bypass, i.e. `# type: ignore`.
any reasons to change it back to `List`?

btw, i thought you want to change from List to Sequence based on slack conversation....

Reply:
Sequence does not allow for indexed access which was a problem downstream. List is more generic